docs/index.rst

```diff
@@ -25,4 +25,5 @@
     doc-src/overview.rst
     doc-src/hardware-description.rst
     doc-src/software-description.rst
+	doc-src/eFPGA-description.rst
```

Comment:
There are either extra spaces or tabs on this line.  `Sphinx` is a finicky tool so it would be best to ensure that this file has no spaces and that line 28 is aligned with the previous lines.